internal/aws/containerinsight/const.go

```diff
@@ -145,6 +146,7 @@ const (
 	// Special type for pause container
 	// because containerd does not set container name pause container name to POD like docker does.
 	TypeInfraContainer = "InfraContainer"
+	TypePodStatus      = "PodStatus"
```

Comment:
We shouldnt need this. `TypePod` above should suffice for these changes.

Reply:
How's that sufficient? prefix is different. TypePod converts to pod_*, not pod_status_*